dashboard/src/components/ListingItem/ListingItem.tsx

```diff
@@ -89,21 +94,45 @@ const ListingItem = ({
     <></>
   );
 
+  const TooltipComponent = ({ children }: PropsWithChildren): JSX.Element => {
+    return (
+      <Tooltip>
+        <div className="flex">
+          <TooltipTrigger className="overflow-hidden">
```

Comment:
I think we probably want an `asChild` here so it doesn't render as an html button